### PR TITLE
Case MUMC+ 

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
A case about our client MUMC+.

Waiting for approval from them

### CODE REVIEW

File: source/cases/nl/mumc.html.markdown.erb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  person: "Michel van Zandvoort"
+  position: "Directeur Maastricht UMC+ Academie"
+facts:
+  branch: Zorg
+  url: https://academie.mumc.nl/
+  since:
+  employees:
+---
+
+**Professionals in de gezondheidszorg helpen met het reflecteren op hun eigen handelen om de zorg van binnenuit te verbeteren. Dat is een van de manieren waarop het Maastricht UMC+ *performance support* toepast binnen Maastricht Universitair Medisch Centrum. Een belangrijk onderdeel hiervan is het werkplekgericht curriculum “klinisch redeneren”.**
```
Comment (sn3p) [inline source/cases/nl/mumc.html.markdown.erb]: `werkplek gericht` I think? (not sure)

File: source/cases/nl/mumc.html.markdown.erb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+De scenario's die hiervoor gebruikt worden, gaan hand in hand met de eisen in [het Kwaliteitspaspoort](/kwaliteitspaspoort/). Door middel van de assessments in de praktijk, wordt de medewerker bekwaam verklaard in zijn kwaliteitspaspoort. Werkplekgerichte curriculum “klinisch redeneren” is onderdeel van een allesomvattende visie op werkplekleren. Deze visie beschrijft:
```
Comment (sn3p) [inline source/cases/nl/mumc.html.markdown.erb]: `Werkplek gerichte`

File: source/cases/nl/mumc.html.markdown.erb [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+- Leren reflecteren op (eigen) handelen waarbij een beroep wordt gedaan op het eigen professioneel handelen;
+- De contextrijke werkomgeving optimaliseren voor leren;
+- Creëren van een ondersteunende leeromgeving.
+
+Hierin is ook het [70:20:10-model](/70-20-10/) terug te vinden, al is dat niet zo verwoord. In al hun communicatie rondom educatie en performance support gebruikt Maastricht UMC+ academie de taal die ook op de werkvloer gebruikt wordt. Michel van Zandvoort, directeur van de Maastricht UMC+ academie: ‘Wanneer verpleegkundigen “klinisch redeneren” horen, dan weten ze meteen ‘Hé, dit is voor mij relevant’.’
+
+Het werkplekgerichte curriculum helpt bij het reduceren van diagnostische fouten. Het maakt de sterke en de te ontwikkelen kanten van de betrokken zorgprofessionals inzichtelijk. Onder- en overschattingen van het eigen kennen en kunnen is heel menselijk, maar kan grote gevolgen hebben. Daarom helpt de Maastricht UMC+ academie om dit inzichtelijk te maken voor hun medewerkers. Ze geven handvaten, zodat medewerkers zich verder kunnen ontwikkelen.
```
Comment (sn3p) [inline source/cases/nl/mumc.html.markdown.erb]: `werkplek gerichte`

File: source/cases/nl/mumc.html.markdown.erb [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
...werknaam voor het werkplek-leren binnen MUMC+ academie. Door middel van werkplek-assessment wil het MUMC+ de cognitieve vermogens van medewerkers verder ontwikkelen. Gerjan Willems, ontwikkelaar en projectleider werkplekleren legt uit: ‘Werkplek-leren is effectief en ontwikkelingsgericht omdat het assessment plaatsvindt in de meest authentieke situatie die er is: de relatie patiënt-medewerker. Van de andere kant kunnen de verletkosten dalen doordat de medewerker niet van de werkplek af hoeft om zich verder in het vak te ontwikkelen.’ Tijdens dagelijkse bezigheden wordt een medewerker gevolgd door een speciaal getrainde collega, een zogenaamde “assessor”. Tijdens het assessment zoveel mogelijk open vragen om de medewerker aan het redeneren te krijgen. De patiënt wordt dan ook integraal betrokken bij zijn eigen assessment. Er worden geen gesloten gevragen gesteld omdat de medewerker dan niet vanuit de patiënt gaat redeneren en analyseren. In de evaluatie van het assessment wordt geen waardeoordeel gegeven, er wordt uitsluitend cognitieve feedback gegeven waarna zowel de medewerker als de assessor reflecteren op de *bedside performance*. Ook de patiënt evalueert zijn eigen assessment.
```
Comment (sn3p) [inline source/cases/nl/mumc.html.markdown.erb]: gesloten **~~ge~~**vragen gesteld

File: source/cases/nl/mumc.html.markdown.erb (hunk 1 above)
Comment (sn3p) [reply source/cases/nl/mumc.html.markdown.erb]: Or is this a title? There are a lot of `Werkplekgericht` instances in here.

File: source/cases/nl/mumc.html.markdown.erb [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+## Bevoegd en bekwaam
+
+In 2007 heeft de Raad van Bestuur van het MUMC+ vastgesteld dat de medewerkers voortaan aantoonbaar bevoegd en bekwaam moeten zijn. ‘Daar is destijds geld beschikbaar voor gesteld en is heel rigide doorgevoerd,’ aldus Michel van Zandvoort. ‘Er zijn toen verscheidene e-learningmodules ontwikkeld, zodat er geen discussie meer nodig was of mensen wel of niet bevoegd en bekwaam moesten zijn. Toch liepen we nog wel tegen een probleem aan. Er kwam een hoop weerstand van de werkvloer wanneer medewerkers voor de derde keer moesten laten zien dat ze de apparatuur goed konden bedienen. Je haalt mensen van de werkvloer om in een e-learning te laten zien dat ze iets kunnen waarbij we weten dat het maken van bijvoorbeeld een e-learning niet automatisch betekent dat de performance van de medewerker verbeterd. Dat pakken we nu beter aan.’
```
Comment (sn3p) [inline source/cases/nl/mumc.html.markdown.erb]: `e-learningmodules` > `e-learning modules` ?

Comment (sn3p) [inline source/cases/nl/mumc.html.markdown.erb]: `bekwaam moesten zijn` > `bekwaam moeten zijn`

Comment (RonaldDijkstra) [reply source/cases/nl/mumc.html.markdown.erb]: truth

**Pull request merged**